### DISCUSSION TITLE
It is not possible to load file

### DISCUSSION THREAD

**Discussion by Agung41** [tahoma2d/tahoma2d#1552] [Q&A]
Hello, my tahoma acting weird, everytime i try to load a scene it wont open and give a warning that says "it is not possible to load file ..... . pli". Critical

My tahoma once crashed, and after that crash i always get this problem. I can still load my previous scene befor it crashed, but cant open newer file, it can open but the drawing , the pli file always corrupt, the bak file of the drawing also corrupt and cant be seen

I tried to open the newer tahoma 2d and the problem still there in the new extracted tahoma

Whats the problem here and how to fix this. Thank you
![IMG_20240822_115948](https://github.com/user-attachments/assets/0579960f-6e00-47bf-b17d-87b0e0847457)

**Comment by manongjohn** [comment]
It's possible the level is corrupted, though I had a similar warning with a PLI file loading a scene, but when I `Replace Level` with the same file it some how loaded anyway.
Did you try reloading that level into the scene manually?
